src/rev_ai/models/__init__.py

```diff
@@ -2,7 +2,6 @@
 """Models"""
 
 from .job import Job
-from .job_options import JobSubmitOptions
 from .job_status import JobStatus
```

Comment:
off topic note: consider adding this to src/rev_ai/__init__.py , it looks like the end user may end up using it and having a convenient import is helpful